mobile/Screen/SubmissionScreen.js

```diff
@@ -1,20 +1,288 @@
+        },
+        title: artworkTitle,
+        description,
+        objects,
+      });
+      setSubmissions((prev) => ([...prev, res.data]));
+      clearInput();
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  const addFile = async (fileURI, fileType) => {
+    console.log(fileURI);
+    setFiles((prev) => ([...prev, { uri: fileURI, type: fileType }]));
+    if (fileType.includes('image')) {
+      setThumbnails((prev) => ([...prev, fileURI]));
+    } else if (fileType.includes('video')) {
+      try {
+        const { videoToImageURI } = await VideoThumbnails.getThumbnailAsync(fileURI);
+        setThumbnails((prev) => ([...prev, videoToImageURI]));
+      } catch (err) {
+        console.error(err);
+      }
+    } else { // Implement default thumbnail in future if we have additional file types (e.g. audio)
+
+    }
+  };
+
+  const pickImage = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    if (!result.canceled) {
+      addFile(result.assets[0].uri, result.assets[0].type);
+    }
+  };
+
+  const openCamera = async () => {
+    const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+
+    if (permissionResult.granted === false) {
+      return;
+    }
+
+    const result = await ImagePicker.launchCameraAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    if (!result.canceled) {
+      addFile(result.assets[0].uri, result.assets[0].type);
+    }
+  };
+
+  const openDocSelector = async () => {
+    const result = await DocumentPicker.getDocumentAsync({
+      multiple: true,
+      copyToCacheDirectory: true,
+    });
+
+    if (result.type === 'success') {
+      addFile(result.uri, result.mimeType);
+    }
+  };
+
+  const getSubmissions = async () => {
+    const result = await axios.get(`${URL}/submissions/get`);
+    setSubmissions(result.data);
+  };
+
+  useEffect(() => {
+    getSubmissions();
+  }, []);
+
+  console.log(thumbnails);
+
   return (
-    <View style={styles.container}>
-      <Text>Submission Screen</Text>
-    </View>
+    <ScrollView>
+      <View style={styles.container}>
+        <View>
+          <Button title="Clear" onPress={clearInput} />
+          <Button title="Image" onPress={pickImage} />
+          <Button title="Camera" onPress={openCamera} />
+          <Button title="File" onPress={openDocSelector} />
+
+          {thumbnails.map((uri, i) => (
+            <Image
+              source={{ uri }}
+              key={i}
+              style={{ width: 200, height: 200 }}
+            />
+          ))}
+          <TextInput
+            styles={styles.input}
+            placeholder="Name"
+            onChangeText={(newName) => setName(newName)}
+            defaultValue={name}
+            ref={nameRef}
```

Comment:
There should be an "on return" property that switches to the next `ref` to change to the next text field